lib/Test/Perl/Critic.pm

```diff
@@ -40,22 +47,16 @@ sub import {
     if ( exists $args{-format} ) { $args{-verbose} = $args{-format}; }
     %CRITIC_ARGS = %args;
 
+    # reset possibly lazy-initialized Perl::Critic
+    $CRITIC_OBJ = undef;
```

Comment:
Without this commit, this might have happened:

    Test::Perl::Critic->import( -profile => 'profile-A' );
    critic_ok('source.pm');
    # now $critic is initialized to P::C instance referring to profile-A
    
    Test::Perl::Critic->import( -profile => 'profile-B' );
    # here $critic hold the reference to P::C instance referring to profile-A
    
    critic_ok('source.pm')
    # the result might not be what you would expect